core/lib/presentation/views/clipper/side_arrow_clipper.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+class SideArrowClipper extends CustomClipper<Path> {
```

Comment:
can you add comments on what your class does? i know the name, but give some information is better I think